applications/FluidDynamicsApplication/custom_processes/shock_capturing_physics_based_process.h

```diff
@@ -353,15 +335,15 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) ShockCapturingPhysicsBasedProcess :
                 const double h_kappa = h_ref * norm_2(r_grad_temp) / std::sqrt(CalculateProjectedInverseMetricElementSize(r_inv_metric_tensor, r_grad_temp) + eps);
 
                 // Thermal sensor (detect thermal gradients that are larger than possible with the grid resolution)
-                const double s_kappa_0 = 1.0;
+                const double s_kappa_min = 0.0;
```

Comment:
Why changing it to 0? As far as I understand, the thermal sensor is shifted to more than 1.0 since from 0.0 to 0.9999 the mesh can resolve the thermal gradients.